webauthn-server-core/src/main/java/com/yubico/webauthn/AssertionRequest.java

```diff
@@ -59,8 +59,10 @@ public class AssertionRequest {
    * <p>If both this and {@link #getUserHandle() userHandle} are empty, this indicates that this is
    * a request for an assertion by a <a
    * href="https://www.w3.org/TR/2021/REC-webauthn-2-20210408/#client-side-discoverable-public-key-credential-source">client-side-discoverable
-   * credential</a>, and identification of the user has been deferred until the response is
-   * received.
+   * credential</a> (passkey), and identification of the user has been deferred until the response
```

Comment:
I would remove the comma before "and"

Reply:
I agree this comma is awkward, but I don't like removing it either. I replaced it with a period and rewrote the subordinate clause a bit instead. :slightly_smiling_face: